index.bs

```diff
@@ -7521,6 +7521,18 @@ of the same shape.
 		[=decibels to linear gain unit|converted to linear
 		units=] and sampled at the time of processing of this
 		block (as [=k-rate=] parameters).
+		
+	1. Let <var>knee end threshold</var> have the value, also
+		[=decibels to linear gain unit|converted to linear
+		units=] after calculation, of
+		{{DynamicsCompressorNode/threshold}} plus
+		{{DynamicsCompressorNode/knee}} sampled at the time of
+		processing of this block (as [=k-rate=] parameters).
```

Comment:
I think I'd break this into two steps:

1. Compute the sum of {{DynamicsCompressorNode/threshold}} and {{DynamicsCompressorNode/knee}}, summing the decibel values.
1. Let |knee end threshold} be the sum in the previous step, [=decibels to linear gain|converted to linear units=].

With this change, I think you can remove the Note below.

WDYT?

Reply:
I think that's a great clarification! It makes it much simpler to understand the steps. A "Note" is an indicator to me that I'm being unclear :).

I'll update the patch in the next few days! Thanks!